.circleci/config.yml

```diff
@@ -159,6 +162,19 @@ jobs:
              account: $AWS_ACCOUNT_PRODUCTION
              profile_name: default
              role: 'LBH_Circle_CI_Deployment_Role'
+  check-code-formatting:
+    executor: docker-dotnet
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install dotnet format
+          command: dotnet tool install -g dotnet-format
+      - run:
+          name: Run formatter check
+          command: dotnet format --dry-run --check
+          environment:
+            PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/root/.dotnet/tools
```

Comment:
Should we save this in Circle CI as an environment variable? Looks pretty long 😄

Reply:
I've removed this variable and installed the tool to a path instead :)